modules/apps/callbacks/types/errors.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+import (
+	errorsmod "cosmossdk.io/errors"
+)
+
+var (
+	ErrNotPacketDataProvider   = errorsmod.Register(ModuleName, 2, "packet is not a PacketDataProvider")
+	ErrCallbackMemoKeyNotFound = errorsmod.Register(ModuleName, 3, "callback memo key not found")
```

Comment:
I think we can drop the reference to memo (its an implementation detail of other applications), maybe we can say something like packet data does not have callback key

Reply:
done